homeassistant/components/tts/__init__.py

```diff
@@ -175,7 +180,7 @@ async def async_setup_platform(p_type, p_config=None, discovery_info=None):
         async def async_say_handle(service: ServiceCall) -> None:
             """Service handle for say."""
             entity_ids = service.data[ATTR_ENTITY_ID]
-            message = service.data.get(ATTR_MESSAGE)
+            message = service.data[ATTR_MESSAGE]
```

Comment:
I think this is the only non-type hint change I have made.
Based on the schema, it looks like ATTR_MESSAGE is compulsory.